node-installer/internal/config/config.go

```diff
@@ -17,6 +17,8 @@ type Config struct {
 	Files []File `json:"files"`
 	// RuntimeHandlerName is the name of the runtime handler (containerd runtime) to create.
 	RuntimeHandlerName string `json:"runtimeHandlerName"`
+	// QemuExtraKernelParams is an additional set of kernel parameters to pass kernels started with QEMU.
+	QemuExtraKernelParams string `json:"QemuExtraKernelParams"`
```

Comment:
```suggestion
	QemuKernelParams string `json:"QemuKernelParams"`
```
Now it's not an "extra" anymore ;D

Reply:
You could argue it is though: The kata shim already passes a bunch of kernel parameters and the parameters here are appended at the end.